packages/cli/src/commands/deploy-v2.ts

```diff
@@ -1,6 +1,6 @@
 import chalk from "chalk";
 import glob from "glob";
-import path, { basename } from "path";
+import path, { basename } from "node:path";
```

Comment:
this is me attempting to fix build issues

I believe vite, etc. bundle in their own polyfill so seems fine to use the node lib directly rather than whatever polyfill we were manually adding as a dep cc @dk1a